.github/workflows/integer_multi_gpu_full_benchmark.yml

```diff
@@ -0,0 +1,184 @@
+# Run all integer benchmarks on an instance with CUDA and return parsed results to Slab CI bot.
+name: Integer multi GPU full benchmarks
```

Comment:
@soonum may I ask you to check this workflow please? I think you already checked the one for tests.